clusterloader2/pkg/measurement/util/wait_for_pods.go

```diff
@@ -88,7 +93,7 @@ func WaitForPods(clientSet clientset.Interface, stopCh <-chan struct{}, options
 			}
 			// We allow inactive pods (e.g. eviction happened).
 			// We wait until there is a desired number of pods running and all other pods are inactive.
-			if len(pods) == (podsStatus.Running+podsStatus.Inactive) && podsStatus.Running == options.DesiredPodCount {
+			if len(pods) == (podsStatus.Running+podsStatus.Inactive) && podsStatus.Running == podsStatus.RunningButNotReady && podsStatus.RunningUpdated == options.DesiredPodCount {
```

Comment:
I don't understand it - why `Running == RunningButNotReady` ?

Reply:
:D I meant to write Running == RunningUpdated to check that all running pods are already updated Fixed